website/docs2/.svelte-kit/generated/nodes/0.js

```diff
@@ -0,0 +1 @@
+export { default as component } from "../../../node_modules/@sveltejs/kit/src/runtime/components/layout.svelte";
```

Comment:
What is the **docs2** folder about?  Looks auto-generated. If it can be auto-generated locally then please remove from this PR and put it into the respective **.gitignore** file accordingly. Otherwise please let me know why it should be here ?